vowpalwabbit/conditional_contextual_bandit.cc

```diff
@@ -489,6 +489,16 @@ void learn_or_predict(ccb& data, multi_learner& base, multi_ex& examples)
         save_action_scores(data, decision_scores);
         VW_DBG(examples) << "ccb "
                          << "slot:" << slot_id << " " << ccb_decision_to_string(data) << std::endl;
+        for (const auto& ex : data.cb_ex)
```

Comment:
since this is done during learn_or_predict, won't the shared features be merged into the actions and slots already by the shared_feature_merger?

Reply:
Yes they will be. This is basically a new way to count features for ccb. The feature count of each slot will include all single features and interactions for each action that slot processes